### ISSUE TITLE
Footnotes text size renders incorrectly with Hugo 0.98.0 or later

### ISSUE THREAD

**Issue by benhsz** [LukasJoswiak/etch#43]
When using Hugo 0.98.0 or later, the footnotes text size is the same size as the main content of the blog post, and fiddling with `main#content section.footnotes` in main.css does nothing.

It seems something changed with Hugo 0.98.0, because reverting to Hugo 0.97.3 or earlier fixes the problem.

**Comment by LukasJoswiak** [comment]
Apologies for the late reply. I cannot reproduce this currently with

```
% hugo version
hugo v0.101.0+extended darwin/amd64 BuildDate=unknown
```

Are you still seeing this behavior?

**Comment by benhsz** [comment]
I'm still seeing it, yeah. The version I'm using locally to work around it is

```
hugo v0.97.3-078053a43d746a26aa3d48cf1ec7122ae78a9bb4 windows/amd64 BuildDate=2022-04-18T17:22:19Z VendorInfo=gohugoio
```

If I use this version

```
hugo v0.101.0-466fa43c16709b4483689930a4f9ac8add5c9f66+extended windows/amd64 BuildDate=2022-06-16T07:09:16Z VendorInfo=gohugoio
```

Then I see the problem again.

I forgot to mention I'm using Firefox 105.0.2 but I'm seeing it with Chrome and Edge as well.

When I visit https://lukasjoswiak.github.io/etch/markdown-syntax-guide/ the footnote at the bottom appears normal, so I'm not sure what's going on exactly.

**Comment by LukasJoswiak** [comment]
Can you run the below commands from a new directory and send a screenshot of what you see?

```
$ hugo new site quickstart
$ cd quickstart
$ git init
$ git submodule add https://github.com/LukasJoswiak/etch.git themes/etch
$ echo '\n[params]\n  copyright = "Copyright © 2020 Your Name"' >> config.toml
$ hugo server -t etch
```

When I run the above and visit localhost:1313 (Hugo v0.104.3, Firefox 105.0) I see

<img width="281" alt="Screen Shot 2022-10-08 at 14 21 40" src="https://user-images.githubusercontent.com/3682510/194728281-8c44f511-f80d-4f30-96f0-29d311b1a550.png">

which looks normal to me. It looks the same on Safari.

**Comment by benhsz** [comment]
That part seems to be unaffected, I've highlighted the exact area here:

![footnote](https://user-images.githubusercontent.com/91887122/194939281-b7a9926d-068d-41b0-9f92-e1830769cd91.gif)

The problem, which I avoid by using an earlier version of Hugo, is that the text in that area renders slightly larger than it should:

![problem](https://user-images.githubusercontent.com/91887122/194939245-16f35b5b-b15e-4d2c-a7f5-d389899450ed.gif)

**Comment by LukasJoswiak** [comment]
Ah my apologies, I misinterpreted your original post. This looks like it was due to how the HTML for footnotes is generated. It used to use a `section` tag and now uses a `div`. I submitted a fix in #48. Let me know if this fix works for you, and I'll merge the linked PR soon.

**Comment by benhsz** [comment]
Yeah, that fixes the problem. Thanks.
